AirBelgie.Migrations/scripts/202503090105_add_user_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE public.users (
```

Comment:
I was planning on creating a UserRepository and doing things with a controller. But this PR started getting big so I've split it out for now.

I'm leaving this here to build upon in the next PR that actually uses the patterns set out here and creates a UserRepository and has a controller that reads from that table.